packages/svelte-urql/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from '@urql/core';
-export * from './operationStore';
-export * from './context';
-export * from './operations';
+export * from './queryStore';
+export * from './mutationStore';
+export * from './subscriptionStore';
+export * from './common';
```

Comment:
why are we exporting everything form common, mainly asking because it contains a few methods that aren't useful to the client

Reply:
I could strip out the functions exports.  IMO it would be good to provide them in case others want to build a custom tailored version.  I don't feel strongly either way.

Reply:
I'd prefer to keep them unexported. They really don't belong to our API surface and all other packages in `@urql/*` basically ensure that their API surface is as minimal as possible to keep them stable. This of course hasn't applied to Svelte as much as this is basically the third iteration, but seen from another perspective, `createPausable`, `fetchProcess`, `fromStore`, and `defaultBaseResult` are all not part of our public API or part of urql's logic. So they're utilities that'd not have much use outside of the core bindings.

There are two sides to this:
1. We'd like to hear from people when they run into limitations, and if they realise they'll have to copy `@urql/svelte` code and modify it, we'll hear from them and be able to adapt
2. These utilities are all not part of what we maintain and if we change behaviour we'd like to ensure that we don't have to continue maintaining them when other implementation details change

Reply:
resolved in https://github.com/FormidableLabs/urql/pull/2370/commits/ac8e9ed0d4bf56d6eac252487b3d7214d7959d1c